### PR TITLE
remove DCIM from temp path patterns.

### LINKED ISSUE #63: DCIM folder is skipped when uploading
### Expected Behaviour

When performing a deep folder upload, the DCIM folder and its descendants should be included.

### Actual Behaviour

The DCIM folder and its contents are skipped entirely.

### Reproduce Scenario (including but not limited to)

`FileSystemUpload` is being used in conjunction with its "Perform deep upload" to upload all files in a local directory to an AEM instance.

#### Steps to Reproduce

* Create a new folder on the local file system.
* Within this new folder, create a folder called `DCIM`.
* Copy one or more images into both of the newly created folders.
* Use code from the library's examples to create a `FileSystemUpload` that uses the `.withDeepUpload(true)` option.
* Using the newly create upload, call `fileUpload.upload(options, ['/path/to/folder/in/first/step']`.
* Note that the folder created in the first step (and all of its images) are uploaded, but the `DCIM` folder is not.

#### Platform and Version

* Mac 11.6 (Big Sur)
* Node.JS version v12.20.1
* aem-upload version 1.1.5

#### Sample Code that illustrates the problem

N/A

#### Logs taken while reproducing problem

N/A

### PR DESCRIPTION
**Pull request by mfrisbey**
## Description

This PR removes `DCIM` from the set of name patterns for paths that excluded from file system uploads. This means that the `DCIM` folder will be included when uploading files. This fixes #63, which was reported by customers using AEM Desktop.

## Related Issue

See #63 

## Motivation and Context

We made this change because customers wanted to include `DCIM` in their uploads, and we couldn't see a technical reason that the folder should be ignored. The pattern has been included in ignore lists for the lifetime of AEM Desktop (several years), so it's been long enough that we're not sure of the original motivation for ignoring it. 

## How Has This Been Tested?

Unit tests and end-to-end tests pass. A script using the changeset in this PR will successfully include the "DCIM" folder when uploading.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- ~[ ] I have added tests to cover my changes.~
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,12 +40,12 @@
     "@babel/register": "^7.9.0",
     "axios-mock-adapter": "^1.18.1",
     "dotenv": "^8.2.0",
-    "eslint": "^8.2.0",
+    "eslint": "^6.8.0",
     "json-loader": "^0.5.7",
     "mime": "^2.4.4",
-    "mocha": "^9.1.3",
+    "mocha": "^6.2.3",
```
Comment (tmathern) [inline package.json:46]: Why the downgrade? If the older version should be kept, I suggest to pin it, so no-one else updates it again later.

File: package.json [hunk 2]
```diff
@@ -40,12 +40,12 @@
     "@babel/register": "^7.9.0",
     "axios-mock-adapter": "^1.18.1",
     "dotenv": "^8.2.0",
-    "eslint": "^8.2.0",
+    "eslint": "^6.8.0",
```
Comment (tmathern) [inline package.json:43]: Why the downgrade? If the older version should be kept, I suggest to pin it, so no-one else updates it again later.

**Review by tmathern — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mfrisbey) [reply package.json:46]: @tmathern The library needs to be able to run on older versions of Node, and newer versions of these dependencies didn't work on those older versions. By "pin," does that mean just changing the version to something like `"mocha": "6.2.3"`?

Comment (tmathern) [reply package.json:46]: Yes, pinning means that you keep that exact version in package.json (up to the Bugfix version).

Comment (mfrisbey) [reply package.json]: @tmathern I updated the dependency versions - will you please take another look? Thanks!

**Review by tmathern — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (tmathern) [reply package.json]: Done.

**Pull request merged**